01_fib/brvishnu92/main_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		fmt.Println(actual)
+		t.Errorf("Unexpected output in main()")
+	}
+
+}
+
+func TestMainOutputPositive(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(7)
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
```

Comment:
Consider using raw strings for readability instead of `"1\n1\n2\n3\n5\n8\n13\n"`
```go
`1
1
2
3
5
8
13
`
```

Reply:
Hi Julia,

I have made the changes requested. But I am not sure what to do next? Do I need to raise a new pull request as my changes arent reflected here? Thanks :-D